django/core/management/commands/migrate.py

```diff
@@ -201,14 +201,17 @@ def handle(self, *args, **options):
 
     def migration_progress_callback(self, action, migration, fake=False):
         if self.verbosity >= 1:
+            import time
             if action == "apply_start":
+                self.start = time.time()
                 self.stdout.write("  Applying %s..." % migration, ending="")
                 self.stdout.flush()
             elif action == "apply_success":
+                elapsed = time.time() - self.start
                 if fake:
-                    self.stdout.write(self.style.MIGRATE_SUCCESS(" FAKED"))
+                    self.stdout.write(self.style.MIGRATE_SUCCESS(" FAKED (%s)" % elapsed))
                 else:
-                    self.stdout.write(self.style.MIGRATE_SUCCESS(" OK"))
+                    self.stdout.write(self.style.MIGRATE_SUCCESS(" OK (%s)" % elapsed))
             elif action == "unapply_start":
                 self.stdout.write("  Unapplying %s..." % migration, ending="")
                 self.stdout.flush()
```

Comment:
I thought about that as well. The only question I have is, do we already show it for a verbosity of 2 or only 3? I normally to run everything with `-v 2`, but the elapsed time is nothing that'd be useful unless I'm debugging something, which I would run with `-v 3` anyway.